modules/oxide-components/src/main/ts/components/menu/Menu.tsx

```diff
@@ -0,0 +1,244 @@
+import { Arr, Fun, Id } from '@ephox/katamari';
+import { forwardRef, useEffect, useMemo, useRef, useState, type FC } from 'react';
+
+import { Icon } from '../../internal/icon/Icon.component';
+import { Bem, Dropdown, KeyboardNavigationHooks } from '../../main';
```

Comment:
I don't think we should import from `main`​ use relative imports for these different things. I think we have a lint rule if this was setup like other libraries. We should really fix that make this `api/Main.ts`​ instead.